proto/spacemesh/v1/smesher_types.proto

```diff
@@ -128,3 +128,17 @@ message PostSetupStatus {
     PostSetupOpts opts = 3; // setup options previously set by the user
     string error_message = 4; // The error message, if the state is STATE_ERROR
 }
+
+message SmeshingStatusRequest {
+    string smesher_id = 1;
+    repeated uint32 epochs = 2; // Epochs to query status for. If empty, returns status for current, next and subsequent.
```

Comment:
atx is always built for next epoch

Reply:
agree, I don't think this request needs to include the epoch, it can only apply to the current/next epoch.

Reply:
I'm not even sure it needs to include the smesher_id (in other words, we may not need this message type at all) - the node it's talking to already knows its smesher ID, and probably wouldn't have data on another smesher's status, would it?

Reply:
yes, can use nodeID instead smesherID

Reply:
I don't think we need `epochs` here either. This endpoint doesn't need to take any args, and it can just return status for the current and next epochs.